docs/reference/cli.md

```diff
@@ -172,6 +172,8 @@ Options:
   -t, --transpile TEXT        Transpile project models to the specified
                               dialect.
   --append-newline            Include a newline at the end of each file.
+  --cast-function             Use the standard CAST function over the ::
```

Comment:
Hmm, not sure on this naming. `--cast-function` makes it sounds like it's for casting functions.

What do you think of `--no-rewrite-casts` ?

Reply:
Yep, seems better. I'll amend, thanks!

Reply:
Done